src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -813,4 +726,22 @@ private boolean isUntrimmed(String value) {
     public static boolean isValidEmailAddress(String email) {
         return StringHelper.isMatching(email, REGEX_EMAIL);
     }
+
+    /////////////////////////////////////
+    // HELPER METHODS FOR INTERNAL USE //////////////////////////
+    // DUPLICATED IN FieldValidatorTest FOR EXTERNAL TESTS USE //
+    /////////////////////////////////////////////////////////////
```

Comment:
Don't need to include this part. Rather, just add header comments to the following two functions.
Also, consider `getPopulated*` instead of `getInterpolated*`.